tests/baselines/reference/arrayLiteralExpressionContextualTyping.errors.txt

```diff
@@ -1,8 +1,8 @@
-tests/cases/conformance/expressions/contextualTyping/arrayLiteralExpressionContextualTyping.ts(8,5): error TS2322: Type '[number, number, number, string]' is not assignable to type '[number, number, number]'.
-  Types of property 'pop' are incompatible.
-    Type '() => string | number' is not assignable to type '() => number'.
-      Type 'string | number' is not assignable to type 'number'.
-        Type 'string' is not assignable to type 'number'.
+tests/cases/conformance/expressions/contextualTyping/arrayLiteralExpressionContextualTyping.ts(8,5): error TS2322: Type '[1, 2, 3, string]' is not assignable to type '[number, number, number]'.

```

Comment:
hm, it would be nicer if this change didn't happen.

Reply:
Why? `1`, `2`, and `3` are verbatim what was written and are all contextually typed by `number`. It's pretty reasonable, IMO.

Reply:
People expect to see types in a type error, and don't think of `1` as a type. It's not the end of the world; it remains pretty clear because this only happens with contextual typing when the value is sitting right there in plain sight.